zne/noise_amplification/folding_amplifier/glorious_folding_amplifier.py

```diff
@@ -30,9 +31,37 @@
 class GloriousFoldingAmplifier(DAGNoiseAmplifier):
     """Interface for folding amplifier strategies."""
 
+    def __init__(self, barriers: bool = True, tolerance: float = 0.05) -> None:
+        self._set_barriers(barriers)
+        self._set_tolerance(tolerance)
+
+    ################################################################################
+    ## PROPERTIES
+    ################################################################################
+    @property
+    def barriers(self) -> bool:
+        """Barriers setter"""
```

Comment:
This is actually a `getter` not a `setter`, but it is also good if you explain what the property means (e.g. if set to True, barriers are added when folding) 🙂